hadoop-ozone/recon/pom.xml

```diff
@@ -18,11 +18,14 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-ozone</artifactId>
-    <version>1.1.0-SNAPSHOT</version>
+    <version>1.1.0</version>
   </parent>
   <name>Apache Ozone Recon</name>
   <modelVersion>4.0.0</modelVersion>
   <artifactId>hadoop-ozone-recon</artifactId>
+  <properties>
+    <pnpm.version>5.18.9</pnpm.version>
```

Comment:
This change is a part of HDDS-5066 which is merged to master. Can we cherry-pick it from the master instead of including it as part of this change?

Reply:
it's cherrypicked from master. Not sure why it's showing up here.

Reply:
Probably because it is not merged to https://github.com/apache/ozone/tree/1.1.0-RC0.

For cherry-picks we do not need to create pull requests. So, you can try to cherry-pick the commits and push to https://github.com/apache/ozone/tree/1.1.0-RC0 directly. 

After that, this PR should only contain this commit - "commit updated ozone version"

Reply:
yup, I will cherrypick it directly in apache ozone, rebase and upload the diffs again.

Reply:
This should all go away. I am cleaning this up. 5069 will go to master via

https://github.com/apache/ozone/pull/2123

This PR would be clean.

Reply:
I will separate out 5069 from this PR. this PR would only contain pom.xml changes for 1.1.0 release.